Accessibility/AriaNotify/explainer.md

```diff
@@ -366,67 +353,118 @@ if (percent < 100) {
+whitespace](https://infra.spec.whatwg.org/#strip-leading-and-trailing-ascii-whitespace)) before the notification is sent
+to the platform API (invalid strings will throw an exception). 
+
+When no `type` is explicitly provided by the content author, the `type` is set to `notify` by
+default.
+
+To specify a `type`, pass the string as the second parameter.  Alternatively, the `type` may be
+expressed in an object form with property `type`. For example: 
+
+```js
+// Notify of a long-running async task starting and ending 
+document.ariaNotify(
+    "Uploading file untitled-1 to the cloud.",
+    "task-progress-started" ); 
+// ... 
+myfile.asyncFileUpload().then( () => { 
+    document.ariaNotify( "File untitled-1 uploaded.", { 
+         type: "task-progress-finished" } ); 
+}); 
+```
+
+Screen readers may allow their users to filter out these task-progress `type`, may make these notifications
+only available at particular verbosity levels, or may replace the output strings with audio cues. 
+
+#### Example 5 - `type`
+```js
+// User has initiated an action which starts a generation process of data.
+// During the status of the generation, a more critical status needs to be
+// sent to the user 
+document.querySelector("#dataStatus")
+        .ariaNotify( "generating content",
+            { "type":"statusUpdate" }); 
+
+document.querySelector("#dataStatus")
+        .ariaNotify( "processing data ", 
+            { "type":"statusUpdate" }); 
+
+document.querySelector("#dataStatus")
+        .ariaNotify( "counting items ",
+            { "type":"statusUpdate" } ); 
+
+document.ariaNotify( " server connection lost ",
+            { "priority":"high",  
+              "type":"serverStatus" } ); 
+```
+
+As content is being generated, the user is informed of that status. When something more serious occurs, such as losing server access, the server error is prioritized above any pending status updates. Screen readers can choose to offer settings for users to manage specific types, like supressing minor `statusUpdate` notifications entirely.
+
+#### Fallback 
+In the case of browsers that do not yet support `ariaNotify`, `type` is dropped entirely when falling back to ARIA live regions. 
+
+#### Open Issue - Predefined types 
+The use of `type` give the screen reader contextual information regarding the notification which allows for
 creative approaches to dispatching the information to their users. The question then arises of whether the API should
-create a predetermined set of `notificationId` names for common/expected scenarios or whether having predefined names is
+create a predetermined set of `type` names for common/expected scenarios or whether having predefined names is
 pointless given no matter the list, it will always fall short. 
 
-Possible examples of predefined `notificationId` could be something like: 
+Possible examples of predefined `type` could be something like: 
```

Comment:
I wonder if we should include some of the possible type categories that the Github team helped put together in https://docs.google.com/document/d/1kNtM_tguuaYKPrhd8cAWfwxj8CJVnWONSZLQp99dtlU/edit?usp=sharing